commands/@.js

```diff
@@ -17,6 +17,28 @@ function isAdmin(){
   return user.id == admin_id
 }
 
+function canRemoveByAngryPoints(value) {
+  if (value > 0) {
```

Comment:
if (value > 0) {
    return false
  }

need to move this code to fun canBeAngry(amount) below if(isAdmin()){ return true }

because admin also can not be angry with positive value

Reply:
ya it is below